### ISSUE TITLE
training results on cifar10

### ISSUE THREAD

**Issue by chunfuchen** [taki0112/ResNeXt-Tensorflow#2]
Are you able to reproduce the accuracy reported in the paper? 

I used another TF wrapper to train ResNeXt29 (8x64d) on the Cifar10/100 dataset, and there are about 2% degradation on both datasets as compared to paper. (I am able to get similar results if I use PyTorch)

Thanks.

**Comment by taki0112** [comment]
ok i will ASAP
